src/targetPacks/assemble.targets

```diff
@@ -1,9 +1,81 @@
 <?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project DefaultTargets="AssembleTargetPack" ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
IMO this is difficult to find and will be confusing in the future if it ends up combining with some other issue. I'd much rather add `Targets="AssembleTargetPack"` to the MSBuild call to be more explicit about what's intended to happen:

https://github.com/dotnet/source-build-reference-packages/blob/fe6afa24c69cc135200bbe1f12a90a11a450dcb0/src/targetPacks/buildTargettingPackages.proj#L34-L36

Reply:
Makes sense.  I updated as suggested.